### DISCUSSION TITLE
How to install air-gapped

### DISCUSSION THREAD

**Discussion by hholst80** [microsoft/pyright#5122] [General]
pip install pyright will not include the required nodeenv. How can I install everything in one go such that the required nodeenv stuff is included?

**Comment by erictraut** [comment]
It sounds like you're using the community-maintained Python wrapper for pyright. Its documentation says that it will install node if necessary. If that's not working for you, you might want to post your question [here](https://github.com/RobertCraigie/pyright-python) so its maintainers can help you out.

**Reply by hholst80** [reply]
My bad I understand that pyright is the javascript/typescript program that is packaged (by the FOSS community) in a separate package.  It is that package that I want some help with.
